Aspire Budgeting/Facilities/ObjectFactory.swift

```diff
@@ -69,11 +69,16 @@ final class ObjectFactory {
     AppDefaultsManager()
   }()
 
+  lazy var googleValidator: GoogleSheetsValidator = {
+    GoogleSheetsValidator()
+  }()
+  
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/aspirebudgetingmobile/aspirebudgeting_ios/blob/11e7508ea9c6ba09d42329eff1bf52e705f4298b/Aspire Budgeting/Facilities/ObjectFactory.swift#L75"></span>Lines should not have trailing whitespace.<br />
<code>trailing_whitespace</code> <code>ObjectFactory.swift:75</code></td>
    </tr>
  </tbody>
</table>